lib/active_record/connection_adapters/postgis_adapter/spatial_table_definition.rb

```diff
@@ -12,14 +12,15 @@ def initialize(types, name, temporary, options, as, adapter)
         # super: https://github.com/rails/rails/blob/master/activerecord/lib/active_record/connection_adapters/abstract/schema_definitions.rb#L320
         def new_column_definition(name, type, options)
           if (info = MainAdapter.spatial_column_options(type.to_sym))
+            if (limit = options.delete(:limit))
+              options.merge!(limit) if limit.is_a?(::Hash)
```

Comment:
@teeparham Not a huge deal, but note that this silently discards `options[:limit]` in the event that it's not a hash.  (It's deleted from `options` regardless of its type, and it's ignored if it's not a hash.)  Is this intentional?  If not, it may be better to raise an error here if `limit` is not a hash.